web/packages/shared/components/UnifiedResources/types.ts

```diff
@@ -237,3 +237,12 @@ export type ResourceViewProps = {
   }[];
   expandAllLabels: boolean;
 };
+
+export type SharedDatabaseServer = {
```

Comment:
What's the difference between a database resource (declared above) and this database server? Are these different things?

Reply:
I think it's worth mentioning in docs for `SharedDatabaseServer`.
Maybe we could also simplify the name? Do we need the "shared" prefix? If you want to differentiate between `teleport` and `shared` types, you can use `import { DatabseServer as SharedDatabaseServer }`.